static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -81,9 +81,11 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                 <% } else if(field.type === "file") { %>
                 <input name="field_<%=field.id%>"
                        type="file"
-                       accept="image/*, .pdf, .doc, .xml, .docx"
+                       accept="image/*, ,video/*, .pdf"
+                       multiple="multiple"
                 <% if (field.required == "1") { %> required <% } %>
                 />
+                <p class="form-info-text">Maximum total size: 6MB.</p>
```

Comment:
I think we need input from @zeek01 on this for two reasons:
- "6MB" does not mean anything unless one knows how large a video get (let alone what MB stands for)
- if people can submit only 3s of video, is this an anti-feature opening ourselves to ridicule?

Reply:
Hm so you suggest just removing the video upload possibility? 
Potentially people could compress video preupload but I accept this is a slim chance

Reply:
Perhaps `Maximum total size: 6MB.` could be: `Maximum file size: 6MB`? 
Okay will ask in channel